app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -204,12 +205,19 @@ class GithubWebhook extends RequestHandler<Body> {
           !file.filename.startsWith('dev/bots/')) {
```

Comment:
PullRequestFile has a deletionsCount, additionsCount, etc, which would make implementing `A PR that only removes code (no modified or added lines) is exempt from the need to add tests.` pretty easy.

Reply:
Oh nice!